ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -114,36 +162,47 @@ export class TransactionDetailsScreen extends React.Component<Props, never> {
       <WalletLayout
         title={I18n.t("wallet.transaction")}
         navigation={this.props.navigation}
-        headerContents={<View spacer={true} />}
+        headerContents={this.getsubHeader()}
         cardType={{ type: CardEnum.HEADER, card: this.props.selectedCard }}
         showPayButton={false}
       >
         <Content scrollEnabled={false} style={WalletStyles.whiteContent}>
           <Grid>
             <Row>
-              <Text bold={true}>{I18n.t("wallet.transactionDetails")}</Text>
+              <H3>{I18n.t("wallet.transactionDetails")}</H3>
+              <Right>
+                <Icon name="io-close" size={variables.iconSizeBase} />
+              </Right>
             </Row>
-            {this.labelValueRow(
-              `${I18n.t("wallet.total")} ${transaction.currency}`,
-              `${transaction.amount}`,
-              "WIDE",
-              false
-            )}
+            <View spacer={true} extralarge={true} />
+
+            <Row>
+              <Text>
+                {`${I18n.t("wallet.total")}  `}
+                <H3>
+                  {" "}
```

Comment:
template literals for these spaces as well -- if need these spaces at all. 

Why are you not using `labelValueRow()`, which serves this exact purpose? I know the invision mockup looks like this, but for the time being I would keep using the "grid-based" layout and possibly update it during a "compliance with the Invision" activity

Reply:
I supposed it was preferred to be adherent to the prototype. Anyway, it would respond to the grid being inserted inside a Row of the same Grid. 
@fgiobergia @cloudify let me know if the previous approach is preferred anyway!